clouddriver-ecs/src/integration/java/com/netflix/spinnaker/clouddriver/ecs/test/CreateServerGroupSpec.java

```diff
@@ -15,57 +15,167 @@
 
 package com.netflix.spinnaker.clouddriver.ecs.test;
 
+import static io.restassured.RestAssured.get;
 import static io.restassured.RestAssured.given;
 import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
 
+import com.amazonaws.services.ecs.AmazonECS;
+import com.amazonaws.services.ecs.model.*;
```

Comment:
nit: Can we add specific imports here instead of *?

Reply:
this was done automatically by intellij (default is to import * after 5 from the same package). I've got my IDE configured [as instructed](https://github.com/spinnaker/clouddriver#developing-with-intellij) for this package so I'm inclined to leave it unless there's some competing community/Java guidance I'm unaware of?

Reply:
Usually it's best to explicitly define the different imports (so that there aren't any accidental usages of the wrong import). That being said, feel free to choose which ever one that works for you.